src/net-questdb-client/Senders/HttpSender.cs

```diff
@@ -212,19 +220,124 @@ private void Build()
     }
 
     /// <summary>
-    /// Creates an HTTP GET request to the /settings endpoint for querying server capabilities.
+    ///     Creates a new HttpClient for the specified address with proper configuration.
     /// </summary>
-    /// <returns>A new <see cref="HttpRequestMessage"/> configured for the /settings endpoint.</returns>
+    /// <param name="address">The address to create a client for.</param>
+    /// <returns>A configured HttpClient for the given address.</returns>
+    private HttpClient CreateClientForAddress(string address)
+    {
+        // Determine the port to use
+        var port = AddressProvider.ParsePort(address);
+        if (port <= 0)
+        {
+            // Use protocol default if no port specified
+            port = Options.protocol switch
+            {
+                ProtocolType.http or ProtocolType.https => 9000,
+                ProtocolType.tcp or ProtocolType.tcps   => 9009,
+                _                                       => 9000,
+            };
+        }
+
+        var host = address.Contains("//")
```

Comment:
I removed the condition and left 
```
var host = AddressProvider.ParseHost(address);
```

and the tests ran OK. I don't think there is a test that exercises the `address.Contains("//") == true` condition